python/requirements.txt

```diff
@@ -1,3 +1,6 @@
 openai==0.27.*
 numpy==1.24.*
-aiofiles>=23.1.0
\ No newline at end of file
+aiofiles>=23.1.0
```

Comment:
I'm surprised to not see these locked down to at least a major version. I've had problems with this in the past but maybe I'm worrying too much. Thoughts?

Reply:
currently is not an issue for our CI, taking down a task to better pin our dependencies